go.mod

```diff
@@ -18,39 +18,41 @@ require (
 	github.com/rancher/wrangler-api v0.6.1-0.20200427172631-a7c2f09b783e
 	github.com/sirupsen/logrus v1.9.0
 	github.com/stretchr/testify v1.8.2
-	k8s.io/api v0.26.1
-	k8s.io/apiextensions-apiserver v0.26.1
-	k8s.io/apimachinery v0.26.1
-	k8s.io/apiserver v0.26.1
+	k8s.io/api v0.27.1
```

Comment:
I think it's better to keep k8s dependencies aligned with core rancher https://github.com/rancher/rancher/blob/release/v2.7/go.mod#L25
Some packages from this operator are imported there so it might be better to share versions.

Reply:
Thanks for review, so we need to downgrade it?

Reply:
I think so it might be easier for us to manage dependencies this way

Reply:
Done